Salada.podspec

```diff
@@ -1,37 +1,20 @@
 Pod::Spec.new do |s|
-  s.name                  = "Salada"
-  s.version               = "2.0"
-  s.summary               = "Salada is OR Mapper for Firebase."
-  s.homepage              = "https://github.com/1amageek/Salada"
-  s.license               = { :type => "BSD" }
-  s.author                = "1amageek"
-  s.social_media_url      = "https://twitter.com/1amageek"
-  s.platform              = :ios
-  s.ios.deployment_target = "8.0"
-  s.ios.framework         = "UIKit"
-  s.requires_arc          = true
-  s.source	              = { :git => "https://github.com/1amageek/Salada.git", :tag => "#{s.version}" }
-  s.source_files          = "Salada/**/*.swift"
-  
-  s.dependency 'Firebase/Core'
-  s.dependency "Firebase/Database"
-  s.dependency "Firebase/Storage"
-
-  s.pod_target_xcconfig = {
 
-      "OTHER_LDFLAGS" => '$(inherited) ' +
-      '-framework "FirebaseCore" ' +
-      '-framework "FirebaseDatabase" ' +
-      '-framework "FirebaseStorage" ' +
-      '-framework "FirebaseInstanceID" ',
+  s.name                    = "Salada"
+  s.version                 = "2.0"
+  s.summary                 = "Salada is OR Mapper for Firebase."
+  s.homepage                = "https://github.com/1amageek/Salada"
+  s.license                 = { :type => 'BSD', :file => 'SaladaFrameworks/LICENSE' }
+  s.author                  = "1amageek"
+  s.social_media_url        = "https://twitter.com/1amageek"
+  s.platform                = :ios
+  s.ios.deployment_target   = "8.0"
+  s.ios.framework           = "UIKit"
+  s.requires_arc            = true
+  s.source                  = { :http => "https://github.com/1amageek/Salada/releases/download/2.1/SaladaFrameworks.zip" }
```

Comment:
It seems there is a problem with the nested classes defined in Salada
I will fix it.